### ISSUE TITLE
Add Spectator Mode

### ISSUE THREAD

**Issue by xavisavvy** [BraedenKilburn/ScrumPoker#11]
I'd love to brainstorm as well to have people who are 'spectators' (like managers, etc.) that would like to see but not participate in the poker.

**Comment by BraedenKilburn** [comment]
Certainly! Introducing a 'spectator' role in the app is a great idea! Right now, our PM has been joining as normal but does not submit a vote, they just manage the vote visibility and clearing of votes.

We could have the users select their role on the home page (i.e., participant or spectator). Participant will be able to see spectators in a separate list or displayed with a distinct label or icon.

We would hide the vote controls from spectators and display a little message to indicate that they're a spectator. We would also update our User model to look something like this:

```typescript
{
  room_id: roomId,
  connection_id: connectionId,
  username: username,
  role: 'participant' | 'spectator',
  point_estimate: null, // Only applicable for participants
}
```

**Comment by xavisavvy** [comment]
I like that <3
